app/test_main.py

```diff
@@ -1,3 +1,31 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_human_age",
+    [
+        (-14, -14, [0, 0]),
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+        ("14", 14, TypeError),
+        (14, "14", TypeError),
```

Comment:
Create another test with wrong types no need to write them all in one.
You need to have two tests:
1. With int values
2. With the wrong types